src/components/EpisodesNav/EpisodesNav.jsx

```diff
@@ -3,21 +3,58 @@ import {
 	data,
 	EpisodesContext as Episodes
 } from "COMPONENTS/Episodes/";
-import { RENDER } from "./";
+import { s, RENDER } from "./";
 
 export default function EpisodesNav(){
-	
+
+	// CONTEXT
+	// -------------------------------
+	const {
+		get: {
+			episodeCount,
+			activeIndex
+		},
+		set: {
+			activeIndex: setActiveIndex
+		}
+	} = useContext(Episodes);
+
+
+	// RENDER
+	// -------------------------------
+	const isFirstEpisodeActive = activeIndex === 0;
+	const isLastEpisodeActive  = activeIndex === episodeCount - 1;
+
 	const pagination = data.map(RENDER.pagination);
+	
+
+	// CLICK HANDLER
+	// -------------------------------
+	function handleNavPressed(e, newIndex){
+		e.preventDefault();
+
+		setActiveIndex(newIndex);
+	}// handleNavPressed
 
 	return (
 		<nav>
 			<button 
+				className={ s.navButton }
 				aria-label="Previous episode."
 				aria-controls="episodes__items episodes__pagination"
+				aria-hidden={ isFirstEpisodeActive }
+				onClick={ (e) => { 
+					handleNavPressed(e, activeIndex - 1) 
+				}}
```

Comment:
as before - this needs to to be `const previous = setIndex.bind(this, activeIndex-1);` and `onClick={previous}` - hang in there, man. Almost there!